assets/src/components/map.tsx

```diff
@@ -117,6 +117,7 @@ const Map = (props: Props): ReactElement<HTMLDivElement> => {
 
   return (
     <div
+      id="vehicle-map"
```

Comment:
What do you think about giving ids a `js-` or `id-` prefix so they don't get mixed up with classes used for CSS?

Reply:
sure. i initially used a `js-map` class but then changed my mind and went with an id, but i'm happy to prefix the id with `id-`